src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -735,17 +735,21 @@ a {
 .anchor {
 	display: none;
 	position: absolute;
-	left: -7px;
+	left: 0;
+	background: none !important;
 }
 .anchor.field {
 	left: -5px;
 }
 .small-section-header > .anchor {
-	left: -28px;
-	padding-right: 10px; /* avoid gap that causes hover to disappear */
```

Comment:
I tried out this change locally, and it all looks good, except that the gap that causes the hover to disappear has cropped up in the new version. Is it possible to get rid of the gap?

Reply:
The gap only appears with headers like `Implementations`, not with enum variants.